lapce-proxy/src/plugin/mod.rs

```diff
@@ -49,7 +49,7 @@ use crate::buffer::language_id_from_path;
 use crate::directory::Directory;
 
 use self::catalog::PluginCatalog;
-use self::psp::{ClonableCallback, PluginServerRpcHandler, RpcCallback};
+use self::psp::{ClosableCallback, PluginServerRpcHandler, RpcCallback};
```

Comment:
I'm pretty sure Clonable is correct here

Reply:
sorry, This is my mistake.I will change this error.